### DISCUSSION TITLE
Driver/Team filtering for audio team radio messages in Live Timing

### DISCUSSION THREAD

**Discussion by SimplyBlu** [f1multiviewer/issue-tracker#440] [Feature requests]
As the AI team radio transcription can be inaccurate at times, being able to listen to the related audio team radio message would be good to have and not having to go scroll through all team radio messages to locate the specific audio message. Suggestion to add a separate driver/team filter for audio radio team messages along with a setting to apply the AI team radio transcription filters apply to the audio team messages section and vice-versa may help solve this.

**Comment by xiphiasphotography** [comment]
Maybe it would be an option in the transcript to add a link to the audio version.
